azext_iot/common/_azure.py

```diff
@@ -241,15 +241,21 @@ def _find_iot_dps_from_list(all_dps, dps_name):
 
 
 def get_iot_central_tokens(cmd, app_id, central_api_uri):
-    def get_event_hub_token(app_id, iotcAccessToken):
+    def formatUrl(url):
+        finalUrl = url
+        if not finalUrl.startswith('https://'):
+            finalUrl = 'https://{}'.format(finalUrl)
+        return finalUrl
+
+    def get_tokens(app_id, iotcAccessToken, central_api_uri):
```

Comment:
nit: can we move away from using closures unless absolutely necessary? They make reading/maintaining code a bit harder